service/s3/internal/customizations/update_endpoint_test.go

```diff
@@ -35,33 +35,6 @@ func Test_UpdateEndpointBuild(t *testing.T) {
 		customEndpoint *aws.Endpoint
 	}{
 		"default endpoint": {
```

Comment:
Removing these tests probably shouldn't be in this PR.

Reply:
fixed